ZFSin/zfs/include/sys/fs/zfs.h

```diff
@@ -805,8 +805,11 @@ typedef struct zpool_load_policy {
  * The location of the pool configuration repository, shared between kernel and
  * userland.
  */
+#if defined(WIN32) || defined(_KERNEL) 
```

Comment:
If win32 AND kernel maybe?

Reply:
ZFSin driver (_KERNEL) writes to the cache file in kernel mode. And zdb reads the cache file in user mode (WIN32). That's why I've made it ||

Reply:
But that means UNIX compile will also use use C:/Windows/ ...

Reply:
should it be like this `#if defined(WIN32) || (defined(_WIN32) && defined(_KERNEL))` ?